docs/podman-commit.1.md

```diff
@@ -0,0 +1,95 @@
+% podman(1) podman-commit - Tool to create an image from a container's changes
```

Comment:
same comment here.  maybe "Create a new image based on the changed container"?

Reply:
fixed